apps/precinct-scanner/src/AppUnhappyPaths.test.tsx

```diff
@@ -166,56 +171,59 @@ test('error from module-scan in accepting a reviewable ballot', async () => {
   })
   fetchMock.getOnce(
     '/scan/status',
-    {
-      status: 'ok',
+    typedAs<GetScanStatusResponse>({
       scanner: ScannerStatus.WaitingForPaper,
       batches: [
         {
           id: 'test-batch',
+          label: 'Batch 1',
```

Comment:
@carolinemodic this is what brought me down this rabbit hole. Many of the places that should have had a `label` didn't, so I added it. Most (all?) of these actually were flagged as type errors, but our CI didn't flag them. We should probably add back a `pnpx tsc --build` step in the build to catch this sort of thing.

Reply:
Ah yeah I noticed a couple that were type errors but didn't stop the build from passing and was wondering about that.